userspace/libsinsp/container.h

```diff
@@ -55,11 +106,19 @@ class sinsp_container_info
 	{
 	}
 
+	static void parse_json_mounts(const Json::Value &mnt_obj, vector<container_mount_info> &mounts);
+
+	container_mount_info *mount_by_idx(uint32_t idx);
+	container_mount_info *mount_by_source(std::string &source);
+	container_mount_info *mount_by_dest(std::string &dest);
+
 	string m_id;
 	sinsp_container_type m_type;
 	string m_name;
 	string m_image;
 	uint32_t m_container_ip;
+	bool m_privileged;
```

Comment:
I was more thinking of a case where you read an old trace file that contains a Docker json blob without the privileged flag and the customer maybe runs sysdig or falco and gets spurious privileged values on that trace, just wanted to make sure that was covered.